### PR TITLE
add crop and contain options, increase full height

### LINKED ISSUE #9162: New System Thumbnail and Image Interface Preview
Source image (landscape):

og-image

Full width preview:

Screen Shot 2021-10-26 at 11 53 47 AM

The problem is that Directus first creates a square cropped thumbnail... and then sets this to `cover` within the interface. This double crops it and looks horrible. We need to add a second system thumbnail that is set to contain (800 wide by 450 tall, ~60% quality, no cropping) and add an Interface option for fit: `cover` (or `crop`?) and `contain` (centered)... both of these would use the new un-cropped system thumbnail.

### PR DESCRIPTION
**Pull request by benhaynes**
closes #9162

Getting some access issues for each thumbnail though... not sure why!

### CODE REVIEW

**Comment by benhaynes** [comment]
closes #9162

File: app/src/interfaces/file-image/file-image.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="image">
+	<div class="image" :class="[width, { crop: crop }]">
```
Comment (rijkvanzanten) [inline app/src/interfaces/file-image/file-image.vue:2]:
```suggestion
	<div class="image" :class="[width, { crop }]">
```

**Review by rijkvanzanten — APPROVED** [review]

**Comment by dteskera** [comment]
hi,
is this merge forgotten? :smile:

**Comment by benhaynes** [comment]
Just taking a while for the team to review... busy times!!

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**